src/main/java/ch/cern/eam/wshub/core/services/administration/impl/ScreenLayoutServiceImpl.java

```diff
@@ -166,6 +168,66 @@ private Map<String, Tab> getCustomGridTabs(InforContext context, String userGrou
         return tabs;
     }
 
+    private Map<String, CustomTab> getCustomTabs(InforContext context, String userGroup, String screenCode) throws InforException {
+        GridRequest gridRequest = new GridRequest("BSFUNC_TBS");
+        gridRequest.getParams().put("parameter.screencode", screenCode);
+        gridRequest.addFilter("type", "HTML", Operator.EQUALS.getValue(), GridRequestFilter.JOINER.OR);
```

Comment:
As you have mentioned yourself, you can pass the tabs as a grid filter to avoid receiving non-utilized tabs